docs/feature-toggles.rst

```diff
@@ -39,7 +39,7 @@ other advanced criteria.
 
 * ``contrib_beta`` - Enable/disable the contributions popup and pages
 * ``kumaediting`` - Enable/disable wiki editing.
-* ``user_interviews`` - Enable/disable user interviews invitation survey banner.
+* ``developer_needs`` - Enable/disable the MDN developer needs survey banner
```

Comment:
You can delete this line now. `developer_needs` is already mentioned above where it belongs.

Reply:
Done!